wiki/Rules/en.md

```diff
@@ -65,3 +65,7 @@ If after several infringements you continue to break the rules, your account may
 Restricted accounts cannot play online, set scores, chat with other users and are for all intents and purposes "banned" from interaction with the community. Restriction for community rule violation can only be removed upon appeal to the support staff after 3 months have passed ([more conditions may apply](/wiki/Help_centre/Account_restrictions#reasons)).
 
 Severe offences that go well beyond the scope of our rules and into the jurisdiction of illegal content as per international common law will result in immediate account termination. This applies nearly exclusively to extremely graphic or otherwise abhorrent offences, such as the distribution of child pornography or other material of abuse.
+
+## Notes
+
+[^macros]: Rebinding multiple keys to K1 or K2 is allowed, as long as a physical key down on any key corresponds to an in-game key down, and a physical key up on any key corresponds to an in-game key up. However, rebinding a key down to a key up or vice-versa on any key is not allowed.
```

Comment:
This reads wrong now.

```suggestion
[^macros]: Rebinding keys to K1 or K2 is allowed, as long as key-down on a specific key performs key-down on a specified game key, and key-up on the same key performs key-up on the same game key. Binding a key-down to a key-up, or having one key perform more than one action is not allow.
```

Reply:
I've committed a slight variation on this, feel free to adjust if you still dislike. (and I'll defer to @Walavouchey for "key-down" vs " key down", but we should be consistent regardless of which we use.)